integration-tests/run-scenario.sh

```diff
@@ -17,16 +17,29 @@
 set -e
 set -o xtrace
 
+#Download the common scripts to working directory
+get_cmn_scripts_dwld(){
+git clone https://github.com/wso2-incubator/test-integration-tests-runner.git
```

Comment:
This clones the master branch. It's ok for now, but when releasing this repo (via git tag), we need to first release test-integration-tests-runner repo. Then, we should checkout the the exact git tag here.

@kavithasub @sashikamw @chaminda